scm/doc/TechGuide/chap_hsd.rst

```diff
@@ -0,0 +1,352 @@
+.. _`chapter: Hierarchical_Physics_Development`:
+
+Hierarchical Physics Development
+================================
+
+Chapter 7 of the CCPP v6 Technical Documentation
+(https://ccpp-techdoc.readthedocs.io/en/v6.0.0/) provides an overview of
```

Comment:
Chapter 7 of this CCPP v6 tech doc seems to be Code Management?

Reply:
This section was added by @dustinswales back when he added the Suite Simulator documentation. I think this is actually supposed to be referencing this chapter itself, not the Tech Doc. I updated it accordingly.

Reply:
I see this as "Chapter %c provides an overview...."

Reply:
Should be fixed now